README.rst

```diff
@@ -142,11 +142,11 @@ the container with ``salt-call``).
 The command ``./run-test.py dev centos7 --salt`` is the same as above but will
 provision the container (eg. calling ``salt-call state.highstate``)
 
-The command ``./run-test.py test centos7`` built a provisioned image
+The command ``./run-test.py test centos7 -- test`` built a provisioned image
 (highstate) and run testinfra tests on it.
 
 Note that you can add any parameters that testinfra (pytest) accept, for
-instance ``./run-test.py test centos7 -- -k idempotence --pdb``.
+instance ``./run-test.py test centos7 -- -k idempotence --pdb test``.
```

Comment:
nop, this is the location of the test directory, no typo in there. This is necessary to avoid pytest discovery of undesired tests such as testinfra's ones.

It would be nicer to have this fixed in run-test.py but it's harder, while fixing the  doc is easy.